### PR TITLE
rename Blazor_net5.0 to BlazorApp

### PR DESCRIPTION
**Pull request by kasperk81**
and upgrade to net7.0

### CODE REVIEW

File: src/mono/wasm/Wasm.Build.Tests/Blazor/MiscTests2.cs [hunk 1]
```diff
@@ -116,20 +116,12 @@ private void BuildNet50Project(string config, bool aot, bool expectError, string
...
-            result.EnsureSuccessful();
-            Assert.Contains("** UsingBrowserRuntimeWorkload: 'false'", result.Output);
-
-            string binFrameworkDir = FindBlazorBinFrameworkDir(config, forPublish: true, framework: "net5.0");
-            AssertBlazorBootJson(config, isPublish: true, binFrameworkDir: binFrameworkDir);
-            // dotnet.wasm here would be from 5.0 nuget like:
-            // /Users/radical/.nuget/packages/microsoft.netcore.app.runtime.browser-wasm/5.0.9/runtimes/browser-wasm/native/dotnet.wasm
-        }
+        result.EnsureSuccessful();
+        Assert.Contains("** UsingBrowserRuntimeWorkload: 'false'", result.Output);
```
Comment (kasperk81) [inline src/mono/wasm/Wasm.Build.Tests/Blazor/MiscTests2.cs:120]:
```suggestion
        Assert.Contains("** UsingBrowserRuntimeWorkload: 'true'", result.Output);
```

**Comment by kasperk81** [comment]
because net 5 is not supported, everything else targeting 5.0 was bumped or from the repo

**Comment by kasperk81** [comment]
because net 5 is not supported, everything else targeting 5.0 was bumped or removed from the repo

**Comment by radical** [comment]
> because net 5 is not supported, everything else targeting 5.0 was bumped or removed from the repo

Sure, but I don't see any harm in keeping them around for now, especially since the corresponding logic in WorkloadManifest.targets hasn't been removed yet. I will be updating our tests to better test different TFMs, and the tests can be dropped as part of that.

**Comment by kasperk81** [comment]
fair enough. then i'll close it.

**Pull request closed without merging**